src/pages/index.tsx

```diff
@@ -161,61 +168,150 @@ function Home() {
             ))}
           </div>
         </div>
-
-        <div className="container mx-auto flex w-auto flex-col">
-          <h2 className="mb-2 text-xl font-medium">
-            Getting started with Meshtastic is as easy as 1, 2, 3!
-          </h2>
-          <ul
-            className="mx-auto relative grid gap-6"
-            style={{
-              gridTemplateColumns: "repeat(auto-fill, minmax(280px, 1fr))",
-            }}
-          >
-            <div className="card">
-              <div className="card__header flex justify-between">
-                <h3>1. Purchase Supported Hardware</h3>
-              </div>
-              <div className="card__body flex justify-center">
-                <p>
-                  Hardware you will want to consider:
-                  <ul>
-                    <li>Radio</li>
-                    <li>Battery</li>
-                    <li>Case</li>
-                    <li>Antenna</li>
-                  </ul>
-                </p>
+        <div className="container mx-auto flex w-auto flex-col gap-8">
+          <h3 className="text-xl font-bold">Getting Started</h3>
+          <div className="flex flex-wrap justify-center gap-6">
+            <div className="flex flex-col items-center">
+              <div className="flex items-center gap-2">
+                <CheckCircleIcon
+                  className="h-8 w-8"
+                  style={{ color: "var(--ifm-color-primary)" }}
+                />
+                <h4 className="text-lg font-bold">Step 1</h4>
               </div>
+              <Link to="/docs/hardware/devices/">
+                <h5 className="font-semibold mt-1">Choose a Device &rarr;</h5>
```

Comment:
Nitpick: anchor tags (link) cannot have a block level (heading) element inside of them.